layouts/notes/list.html

```
@@ -1,28 +1,32 @@
-            <header>
-                <time datetime='{{ .Date.Format "2006-01-02" }}' aria-label="Date: {{ .Date.Format (default "2006-01-02" .Site.Params.dateFormat) }}">
-                    {{ .Date.Format "Jan 02, 2006" }}
+        <article class="feed-item">
+            <header class="feed-item-header">
+                <time class="feed-date" datetime='{{ .Date.Format "2006-01-02" }}'>
+                    {{ .Date.Format (default "2006-01-02" .Site.Params.dateFormat) }}
                 </time>
-                 <a href="{{ .RelPermalink }}" title="Permalink to this note">#</a>
-                 <button class="tinylytics_kudos" data-path="{{ .RelPermalink }}"></button>
+                <a href="{{ .RelPermalink }}" title="Permalink to this note">#</a>
+                <button class="tinylytics_kudos" data-path="{{ .RelPermalink }}"></button>
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)